mlrun/model_monitoring/controller.py

```diff
@@ -801,7 +801,11 @@ def push_regular_event_to_controller_stream(self) -> None:
         logger.info("Starting monitoring controller chief")
         applications_names = []
         endpoints = self.project_obj.list_model_endpoints(
-            tsdb_metrics=False, mode=mm_constants.EndpointMode.REAL_TIME
+            tsdb_metrics=False,
+            modes=[
+                mm_constants.EndpointMode.REAL_TIME,
+                mm_constants.EndpointMode.BATCH_LEGACY,
+            ],
```

Comment:
What will happen after 1.9.x -> 1.10.0 upgrade before upgrading the controller?

Reply:
as far as I know as part of the upgrade you have to update the mm infra (including the controller image) as well. Anyway, even if you didn't, I don’t think it really matters here, since the controller will recognize it’s a batch model endpoint from the `endpoint_type` and then just check if there’s new data in the parquet files. There is no difference in that case between `batch_legacy` and new `batch` ep when you are using a 1.9 controller.